src/ValidatorTicketPricer.sol

```diff
@@ -0,0 +1,212 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.8.0 <0.9.0;
+
+import { PufferOracleV2 } from "puffer/PufferOracleV2.sol";
+import { AccessManaged } from "openzeppelin/access/manager/AccessManaged.sol";
+import { IValidatorTicketPricer } from "./interface/IValidatorTicketPricer.sol";
+
+/**
+ * @title Validator Ticket Pricer
+ * @notice This contract manages the pricing of validator tickets based on MEV payouts and consensus rewards.
+ * @dev Uses PufferOracleV2 for price updates and inherits access control from AccessManaged.
+ */
+contract ValidatorTicketPricer is AccessManaged, IValidatorTicketPricer {
+    uint256 internal constant _BPS_DECIMALS = 1e4; // 100%
+
+    PufferOracleV2 internal immutable _ORACLE;
+
+    // slot 0
+    uint16 internal _dailyMevPayoutsChangeToleranceBps;
+    uint16 internal _dailyConsensusRewardsChangeToleranceBps;
+    uint16 internal _discountRateBps;
+
+    // slot 1
+    uint128 internal _dailyMevPayouts;
```

Comment:
We can pack everything in one slot
3x16 + 2x104 = 256

type(104).max = 20282409603651670423947251286015 = 20282409603651 ETH
I doubt that the daily rewards will go that high.